### DISCUSSION TITLE
How to enable link to privacy policy in the embedded iframe?

### DISCUSSION THREAD

**Discussion by ashishb** [umputun/remark42#1816] [Q&A]
How can I enable a link to the privacy policy in the `remark42`'s embedded iframe?

I can always add it explicitly to my Hugo website but I thought there might be a cleaner way.

**Comment by paskal** [comment]
If I understand your question correctly, you are looking for `/web/privacy.html` path, like `https://demo.remark42.com/web/privacy.html`. That's what I link on the website to meet Google requirement of having the privacy policy.

**Comment by ashishb** [comment]
@paskal Yeah exactly. Is there a way to enable it in the embedded `iframe`?

**Comment by paskal** [comment]
Not yet, but I think it should be possible. I'll add it next to "Powered by Remark42" in the bottom right corner as an option.
<img width="872" alt="image" src="https://github.com/user-attachments/assets/803e9a26-0404-4a31-beaa-558f1ac6b141">

**Comment by umputun** [comment]
I'm a little puzzled about the reason. Remark42 is not an externally hosted system, and this privacy policy pertains to the remark42.com site, which has very little to do with your site that serves the actual commenting engine.

I assume the reason for this request is some bizarre Facebook or Google requirement during OAuth2 registration. From my experience, it can be a link to any place. When Facebook asked me about "where the link on your site" I told them, "well, this is a third-party system; see https://remark42.com/privacy/" and they accepted this answer.

Anyway, as I mentioned, this privacy policy is not about the Remark42 you embedded on your site, but rather about the remark42.com site itself. There is no technical way I can guarantee that your site operates the same as remark42.com from the privacy point of view, so the link to this existing policy makes no sense. Also, the policy mentions me as the operator and remark42.com as the site.
